src/render/html_render.py

```diff
@@ -2,8 +2,60 @@
     HorizontalRule, ListParagraph, QuoteParagraph, BoldBlock, ItalicBlock, \
     ImgBlock, LinkBlock, CodeBlock, FencedCodeBlock, StrikethroughBlock, ListWrapper
 
+
 def render(root):
-    return _dfs(root)
+    rendered_child = []
+    for paragraph in root.children:
+        rendered_child.append(_render_a_paragraph(paragraph))
+    return "<html>{}</html>".format("\n".join(rendered_child))
+
+def _render_a_paragraph(paragraph):
+    if isinstance(paragraph, HorizontalRule):
+        return "<hr/>"
+    if isinstance(paragraph, FencedCodeBlock):
+        return "<code>{}</code>".format(paragraph.content())
+    if isinstance(paragraph, HeaderParagraph):
+        return "<h{0}>{1}<h{0}>".format(paragraph.level(), paragraph.content())
+
+    _format = ""
+    if isinstance(paragraph, TextParagraph):
+        _format = "<p>{}</p>"
+    if isinstance(paragraph, ListWrapper):
+        if paragraph.is_ordered():
+            _format = "<ol>\n{}</ol>"
```

Comment:
why do we specifically need `\n` here?

Reply:
for formatting... but its not really necessary here.